docker/Dockerfile

```diff
@@ -1,25 +1,32 @@
+# https://github.com/rust-lang/docker-rust-nightly/blob/master/buster/Dockerfile
+FROM buildpack-deps:stable-curl AS builder
+
 # Install Rust nightly
 ENV RUSTUP_HOME=/usr/local/rustup \
     CARGO_HOME=/usr/local/cargo \
     PATH=/usr/local/cargo/bin:$PATH
+RUN apt update
```

Comment:
Making apt update more explicit from the `Dockerfile` rather than inside custom script `install_rust.sh`.